File: .github/actions/test-reporter/reports/dart/test/main_test.dart
import 'package:darttest/main.dart';
import 'package:test/test.dart';
import 'dart:io';

void main() {
  group('Test 1', () {
    test('Passing test', () {
      expect(1, equals(1));
    });

    group('Test 1.1', () {
      test('Failing test', () {
        expect(1, equals(2));
      });

      test('Exception in target unit', () {
        throwError();
      });
    });
  });

  group('Test 2', () {
    test('Exception in test', () {
      throw Exception('Some error');
    });
  });

  print('Hello from the test');
}

File: .github/actions/test-reporter/reports/jest/__tests__/main.test.js
const lib = require('../lib/main')

describe('Test 1', () => {
  test('Passing test', () => {
    expect(true).toBeTruthy()
  });

  describe('Test 1.1', () => {
    test('Failing test', () => {
      expect(false).toBeTruthy()
    });

    test('Exception in target unit', () => {
      lib.throwError();
    });
  });
});

describe('Test 2', () => {
  test('Exception in test', () => {
    throw new Error('Some error');
  });
});
